networking/eth2/src/test/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/methods/BlobSidecarsByRangeListenerValidatingProxyTest.java

```diff
@@ -361,38 +331,6 @@ void firstBlobSidecarIndexInNextBlockIsNotZero() {
                 .describe());
   }
 
-  @Test
```

Comment:
why `missedBlobSidecarIndex` is gone?

Reply:
Removed test missedBlobSidecarIndex in BlobSidecarsByRangeListenerValidatingProxyTest is the same as blobSidecarIndexIsInTheSameBlockButNotNext